scripts/linux/runE2ETest.sh

```diff
@@ -0,0 +1,617 @@
+#!/bin/bash
+
+###############################################################################
+# This script is used to streamline running E2E tests for Linux.
+###############################################################################
+set -e
+
+function clean_up() {
+    print_highlighted_message "Clean up"
+
+    echo "Stop IoT Edge services"
+    sudo systemctl stop iotedge.socket iotedge.mgmt.socket || true
+    sudo systemctl kill iotedge || true
+    sudo systemctl stop iotedge || true
+
+    echo "Remove IoT Edge and config file"
+    sudo apt-get purge libiothsm-std --yes || true
+    sudo rm -rf /var/lib/iotedge/
+    sudo rm -rf /var/run/iotedge/
+    sudo rm -rf /etc/iotedge/config.yaml
+
+    echo "Do docker system prune"
+    sudo docker system prune -af || true
+}
+
+function create_iotedge_service_config {
+    print_highlighted_message "Create IoT Edge service config"
+    sudo mkdir /etc/systemd/system/iotedge.service.d/ || true
+    sudo bash -c "echo '[Service]
+Environment=IOTEDGE_LOG=edgelet=debug' > /etc/systemd/system/iotedge.service.d/override.conf"
+}
+
+function get_image_architecture_label() {
+    local arch=$(uname -p)
```

Comment:
It's not necessary for `local` statements, but I still prefer all `$()` to be inside `""`, so `local arch="$(uname -p)"`